timemachine/fe/mle.py

```diff
@@ -212,3 +214,103 @@ def infer_node_vals_and_errs(
     dg_err = bootstrap_estimates.std(0)
 
     return dg, dg_err
+
+
+def infer_node_vals_and_errs_networkx(
+    nx_graph: nx.Graph,
+    edge_diff_prop: str,
+    edge_stddev_prop: str,
+    node_val_prop: str,
+    node_stddev_prop: str,
+    ref_nodes: Sequence[Any],
+    ref_node_val_prop: str,
+    ref_node_stddev_prop: str,
```

Comment:
opinionated super nit: may consider reordering these so that "read only" things are grouped and appear first `edge_diff_prop, edge_stddev_prop, ref_nodes, ref_node_val_prop, ref_node_stddev_prop`, and then the writable properties appear second, possibly with defaults `node_val_prop = "mle_dg", node_stddev_prop = "mle_dg_err"` or similar. (haven't thought carefully about the specific default names)

Reply:
I like this suggestion. Applied in https://github.com/proteneer/timemachine/pull/987/commits/6c2a41add7f35017eb05520e82679172b502acf7, using `"inferred_dg"` and `"inferred_dg_stddev"` (open to alternatives).